src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml

```diff
@@ -33,79 +33,83 @@
                                     }
                                     else
                                     {
-                                        @Html.NavigationRouteLink("View", "View evidence note with reference " + @note.ReferenceId, "Scheme_evidence_default", new { action = "DownloadEvidenceNote", controller = "ManageEvidenceNotes", pcsId = Model.OrganisationId, evidenceNoteId = @note.Id, selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear }, null)
+                                        <td class="govuk-table__cell"> </td>
                                     }
+                                    <td class="govuk-table__cell">@note.Status</td>
+                                    <td class="govuk-table__cell">
+                                        @if (@note.Type == NoteType.Transfer)
+                                        {
+                                            @Html.NavigationRouteLink("View", "View transfer note " + @note.ReferenceId, note.SchemeViewRouteName, new { pcsId = Model.OrganisationId, evidenceNoteId = @note.Id, selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear, redirectTab = ManageEvidenceNotesDisplayOptions.ViewAndTransferEvidence.ToDisplayString(), page = Model.EvidenceNotesDataList.PageNumber }, null)
+
+                                            if (note.DisplayEditLink)
+                                            {
+                                                <br />
+                                                @Html.NavigationRouteLink("Edit", "Edit returned transfer note " + @note.ReferenceId, "Scheme_edit_transfer", new { pcsId = Model.OrganisationId, evidenceNoteId = @note.Id, selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear }, null)
+                                            }
+                                        }
+                                        else
+                                        {
+                                            @Html.NavigationRouteLink("View", "View evidence note with reference " + @note.ReferenceId, "Scheme_evidence_default", new { action = "DownloadEvidenceNote", controller = "ManageEvidenceNotes", pcsId = Model.OrganisationId, evidenceNoteId = @note.Id, selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear, page = Model.EvidenceNotesDataList.PageNumber }, null)
+                                        }
+                                    </td>
+                                </tr>
+                            }
+                        }
+                        else
+                        {
+                            <tr>
+                                <td colspan="5">
+                                    <p>There are currently no evidence notes to view or transfer </p>
+                                </td>
+                            </tr>
+                            <tr>
```

Comment:
the AA screen has a _ViewAllNotesEmptyPartial for the common error for these tabs. Not sure if we can do something similar here?